pkg/daemon/config.go

```diff
@@ -107,43 +107,45 @@ func tryToLoadOldConfig(nodeProfilesJson []byte) ([]ptpv1.PtpProfile, bool) {
 }
 
 func (output *ptp4lConf) populatePtp4lConf(config *string) error {
```

Comment:
it will be helpful to add a sample config string as comments of the function

Reply:
Added a comment about what it does.  I don't think a sample config string would fit well here because of how large it is, and means that we need to keep it updated whenever we make changes to configs.